cvat/apps/engine/views.py

```diff
@@ -2286,6 +2319,8 @@ def _import_annotations(request, rq_id, rq_func, db_obj, format_name,
                 exc_info = exc_info.replace(import_error_prefix + ': ', '')
                 return Response(data=exc_info,
                     status=status.HTTP_400_BAD_REQUEST)
+            elif any([i in exc_info for i in ("DatumaroError", "DatasetImportError", "DatasetNotFoundError")]):
```

Comment:
These exceptions are not wrapped into `CvatImportError` because they occur in `Dataset.import_from` and using the method as I can see is not handled by `DatumaroError`/`DatasetImportError`/`DatasetNotFoundError` anywhere. ([1](https://github.com/opencv/cvat/blob/8ad485be560e6bbf6acedd55f359e7eca85433d3/cvat/apps/dataset_manager/formats/coco.py#LL30C30-L30C30), [2](https://github.com/opencv/cvat/blob/8ad485be560e6bbf6acedd55f359e7eca85433d3/cvat/apps/dataset_manager/formats/cvat.py#L1442)...).

This bug was also fixed.